app/controllers/idv_controller.rb

```diff
@@ -5,6 +5,7 @@ class IdvController < ApplicationController
   before_action :confirm_two_factor_authenticated
   before_action :confirm_idv_needed, only: [:fail]
   before_action :profile_needs_reactivation?, only: [:index]
+  before_action :sp_context_needed?, only: [:index]
```

Comment:
Since this only applies to `:index`, and only the root `IdvController`, what would happen if I navigate to `/verify/doc_auth` or `/verify/doc_auth/welcome` ? Or are we primarily concerned with the base `/verify` path?

Reply:
Opened #4543 to block all {{/verify/*}} paths (which proved to be a bad idea...)